src/freenet/clients/http/wizardsteps/BROWSER_WARNING.java

```diff
@@ -104,6 +99,32 @@ public void getStep(HTTPRequest request, PageHelper helper) {
 		        new String[] { "submit", "next", NodeL10n.getBase().getString("Toadlet.next")});
 	}
 
+	public List<String> oldBrowserWarnings(
```

Comment:
Argh. I really don’t know when to stop nitpicking, I’m sorry. 😢 Feel free to start ignoring comments at any convenient point.

* Inconsistent indentation.
* Parameter order should be modified so that more important parameters come first, in this case `isOldFirefox` seems to be more important than the other ones.
* And again I would try to exit this method as early as possible, i.e. if `!isOldFirefox` I’d return an empty list (convenience method `Collections.emptyList()` exists) and only then start creating the list and adding stuff to it. This also keeps indentation levels in the method itself down for larger parts.